person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :age, :name
+
+  def initialize(age, name)
+    @id = DateTime.now
+    @name = name
+    @age = age
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age?
+  end
+
+  private :of_age
+  public :can_use_services
```

Comment:
- [**Optional**] I don't think you explicitly need to set this method as public. In Ruby, a method is by default public if not under the `private` keyword.